internal/pkg/cli/env_delete.go

```diff
@@ -169,9 +172,23 @@ func (o *deleteEnvOpts) Execute() error {
 		return err
 	}
 	o.prog.Stop(log.Ssuccessf(fmtRetainEnvRolesComplete, o.name))
+
 	if err := o.deleteStack(); err != nil {
+		o.prog.Stop(log.Serrorf("Failed to delete resources for the %q environment\n", o.name))
+		return err
+	}
+
```

Comment:
did we have to `Start` in order to `Stop`? I'm not sure about what happens if we `Stop` a `prog` displayer that hasn't been `Start`ed yet.

```suggestion
        o.prog.Start(fmt.Sprintf("Deleting resources for the %q environment\n", o.name))
	if err := o.deleteStack(); err != nil {
		o.prog.Stop(log.Serrorf("Failed to delete resources for the %q environment\n", o.name))
		return err
         }
         o.prog.Stop(log.Ssuccessf("Deleted resources for the %q environment\n", o.name))
```